en/app.ftl

```diff
@@ -330,6 +338,25 @@ premium-promo-use-cases-gaming-body-2 = Worried about other gamers or trolls lea
 
 premium-promo-pricing-free-price = Your current Free plan
 
+## The Premium waitlist page
+
+waitlist-heading = Join the { -brand-name-relay-premium } Waitlist
+waitlist-lead = Get notified when { -brand-name-firefox-relay-premium } is available for your region.
+waitlist-control-required = Required
+waitlist-control-email-label = What is your email address?
+# Please only translate `yourname`; example.com is an actual example domain that is safe to use.
+waitlist-control-email-placeholder = yourname@example.com
+waitlist-control-country-label = What country do you live in?
+waitlist-control-locale-label = Select your preferred language.
+waitlist-submit-label = Join the Waitlist
+# Variables:
+#  $joinButtonLabel (string): the value of waitlist-submit-label
+waitlist-privacy-policy-agree = By clicking “{ $joinButtonLabel }”, you agree to our <a>Privacy Policy</a>.
```

Comment:
You should be able to use an actual message reference here? Please verify that it works as expected though.

```suggestion
waitlist-privacy-policy-agree = By clicking “{ waitlist-submit-label }”, you agree to our <a>Privacy Policy</a>.
```

Reply:
Ha, of course I can, that makes much more sense! It does work as expected, thanks :)